src/odemis/gui/cont/project.py

```diff
@@ -79,6 +83,8 @@ def _add_project(self, _, name=None):
         logging.debug("Creating new project %s.", name)
         colour = FASTEM_PROJECT_COLOURS[(num - 1) % len(FASTEM_PROJECT_COLOURS)]
         project_ctrl = FastEMProjectController(name, colour, self._tab_data_model, self._project_list, self._viewport)
+        project_ctrl.panel._header.Bind(wx.EVT_LEFT_UP, self._on_project_panel_header)
```

Comment:
Why doesn't the `FastEMProjectController` take care of binding `EVT_LEFT_UP`? This way, it'd be much easier for it to know which `project_ctrl` is clicked on (ie, `self`), and  `_on_project_panel_header()` would be a much simpler function.

Reply:
Its a temporary workaround for now. `_on_project_panel_header` make sure the clicked `project_ctrl` is not collapsed and all other `project_ctrl` in `project_ctrls` dict are collapsed. This forces the user to just work on a single project at a time where any new shape creation will be added to the not collapsed project's ROA. In the VolumeEM Project Manager the project selection will be from a drop-down list.